scripts/ast-patches/utils.ts

```diff
@@ -8,6 +8,10 @@ export const is$wModule = (
   ts.isModuleDeclaration(statement) &&
   statement.name.text === Constants.$w_MODULE_NAME;
 
+export const isGlobalModule = (
+  statement: ts.Statement
+): statement is ts.ModuleDeclaration =>
+  ts.isModuleDeclaration(statement) && statement.name.text === "global";
```

Comment:
can you explain what this global is?

Reply:
we need to wrap $w namespace with global in order for it to be available without importing it.